source/renderer/app/components/wallet/paper-wallet-certificate/PasswordChoiceDialog.js

```diff
@@ -0,0 +1,257 @@
+import DialogBackButton from '../../widgets/DialogBackButton';
+import ReactToolboxMobxForm from '../../../utils/ReactToolboxMobxForm';
+import { isValidWalletPassword, isValidRepeatPassword } from '../../../utils/validations';
+import globalMessages from '../../../i18n/global-messages';
+import styles from './PasswordChoiceDialog.scss';
+
+import showPasswordIcon from '../../../assets/images/show-pass-ic.inline.svg';
+import hidePasswordIcon from '../../../assets/images/hide-pass-ic.inline.svg';
+
+const messages = defineMessages({
+  headline: {
+    id: 'paper.wallet.create.certificate.passwordChoice.dialog.headline',
+    defaultMessage: '!!!Choose your password',
+    description: 'Headline for the "Paper wallet create certificate password choice dialog".'
+  },
+  subtitle: {
+    id: 'paper.wallet.create.certificate.passwordChoice.dialog.subtitle',
+    defaultMessage: '!!!A password you chose will be used to shield your recovery prase on the certificate. You will not be able to restore your wallet without this password.',
+    description: '"Paper wallet create certificate password choice dialog" subtitle.'
+  },
+  passwordComplexityText: {
+    id: 'paper.wallet.create.certificate.passwordChoice.dialog.password.ComplexityText',
+    defaultMessage: '!!!Note that password needs to be at least <strong>7 characters long</strong>, and have at least <strong>1 uppercase character</strong>, <strong>1 lowercase character</strong> and <strong>1 number</strong>.',
+    description: '"Paper wallet create certificate password choice dialog" password complexity text.'
+  },
+  passwordLabel: {
+    id: 'paper.wallet.create.certificate.passwordChoice.dialog.password.label',
+    defaultMessage: '!!!Enter password',
+    description: '"Paper wallet create certificate password choice dialog" password label.'
+  },
+  passwordHint: {
+    id: 'paper.wallet.create.certificate.passwordChoice.dialog.password.hint',
+    defaultMessage: '!!!Type password',
+    description: '"Paper wallet create certificate password choice dialog" password hint.'
+  },
+  repeatPasswordLabel: {
+    id: 'paper.wallet.create.certificate.passwordChoice.dialog.repeatPassword.label',
+    defaultMessage: '!!!Repeat password',
+    description: '"Paper wallet create certificate password choice dialog" repeat password label.'
+  },
+  repeatPasswordHint: {
+    id: 'paper.wallet.create.certificate.passwordChoice.dialog.repeatPassword.hint',
+    defaultMessage: '!!!Repeat password',
+    description: '"Paper wallet create certificate password choice dialog" repeat password hint.'
+  },
+  keepPasswordSecuredConfirmationLabel: {
+    id: 'paper.wallet.create.certificate.passwordChoice.dialog.password.keepConfirmation',
+    defaultMessage: '!!!I understand the importance of the password and I will keep it secure.',
+    description: '"Paper wallet create certificate password choice dialog" password keep confirmation.'
+  },
+  printButtonLabe: {
```

Comment:
`l` is missing in `Label`.

Reply:
Good catch @darko-mijic!
@tomothespian can you please solve this? `printButtonLabe` -> `printButtonLabel`

Reply:
Of course.